src/services/identity/UsersService.ts

```diff
@@ -2,24 +2,40 @@ import { Op, ModelStatic } from "sequelize"
 import { Sequelize } from "sequelize-typescript"
 import { RequireAtLeastOne } from "type-fest"
 
-import { Repo, Site, User, Whitelist, SiteMember } from "@database/models"
+import { Otp, Repo, Site, User, Whitelist, SiteMember } from "@database/models"
+import { BadRequestError } from "@root/errors/BadRequestError"
+import { milliSecondsToMinutes } from "@root/utils/time-utils"
 import SmsClient from "@services/identity/SmsClient"
 import TotpGenerator from "@services/identity/TotpGenerator"
 import MailClient from "@services/utilServices/MailClient"
 
+import OtpService from "./OtpService"
+
+const { OTP_EXPIRY } = process.env
+
+const PARSED_EXPIRY = parseInt(OTP_EXPIRY!, 10) ?? undefined
+
+const MAX_NUM_OTP_ATTEMPTS =
+  parseInt(process.env.MAX_NUM_OTP_ATTEMPTS!, 10) ?? 5
```

Comment:
the ts complain here is correct actually cos these are env vars and are not guaranteed to exist... 

we'd have to write ugly code `parseInt(OTP_EXPIRY || "", 10) ?? undefined` but i think it's ok

Reply:
Yea, and this brings me back to the other point in the virus scan PR - we need to ensure env vars are all present at time of startup (in index.ts) so that we can confidently use `!` here

I will add the ugly code suggestion for now haha

Reply:
think we should use `convict` for env vars tbh and we have a (super) old PR up for this; i think we suffer from having an abundance of env vars, and we don't sanitise them, both of which might be solved w/ `convict`

Reply:
nit: since we are currently not using `convict` or some code to ensure all env vars are loaded up in `index.js` could we add throw error if such a env var doesnt exist?

Reply:
@kishore03109 Am going to add the over arching solution - to check env vars at startup real soon as its pretty important. Will do up a PR next week

Reply:
file issue